docs/list.json

```diff
@@ -998,6 +998,41 @@
 
 		}
 
+	},
+
+	"kr": {
+	
+		"매뉴얼": {
+	
+			"시작하기": {
+				"장면 만들기": "manual/kr/introduction/Creating-a-scene",
```

Comment:
You are still using `kr` in those links.

Reply:
sorry.
modified now